src/endpoints/user-status/data/dtos/userstatus.dto.ts

```diff
@@ -0,0 +1,18 @@
+export type GetUserStatustDto = {
```

Comment:
Conventionally, we use the suffix `RequestDto`.

Can we rename this to `GetUserStatustRequestDto`?